CommonCoreLegacy/src/test/java/com/skedgo/android/common/util/TransportModeUtilsTest.java

```diff
@@ -123,7 +124,7 @@
     final TransportMode mode = new TransportMode();
     mode.setDarkIcon("some-dark-icon");
     assertThat(getDarkIconUrlForTransportMode(resources, mode))
-        .isEqualTo("https://tripgo.skedgo.com/satapp/modeicons/android/xhdpi/ic_transport_some-dark-icon.png");
+        .isEqualTo(Server.ApiTripgo.getValue() + "modeicons/android/xhdpi/ic_transport_some-dark-icon.png");
   }
 
   @Test public void getDarkIconUrlForTransportMode_nonNullTransportModeButNullIconId() {
```

Comment:
@elagarrigue Previously `"https://tripgo.skedgo.com/satapp/` was being used to intentionally prevent a case where `Inflationary` was changed with an invalid name. For example, if somebody somehow changes `Inflationary` into `"https://tripgo.skedgo.com/satapp//"` (incorrect) instead of `"https://tripgo.skedgo.com/satapp/"` (correct), these tests will not pass and thus it'd notify us about the buggy change.

Wdyt if we use `"https://api.tripgo.com/v1/...` instead of `Server.ApiTripgo.getValue() + "`?

Reply:
Yes, I see the reason. I'll change it now